src/Conversion/ONNXToTorch/NN/PaddingOp.cpp

```diff
@@ -94,8 +94,17 @@ class ONNXConstantPadNdOpToTorchLowering : public OpConversionPattern<ONNXPadOp>
 
     // Reading the ONNX side pads values and store in the array.
     std::vector<APInt> intValues;
-    for (auto n : denseAttr.getValues<APInt>())
+    bool paddingNeeded = false;
```

Comment:
The code looks absolutely amazing, but I expected a `FileCheck` test to go with it ;-)